src/views/v2/list/nav.php

```diff
@@ -12,10 +12,15 @@
  * @version TBD
  *
  */
+
+if ( ! tribe_has_previous_event() || ! tribe_has_next_event() ) {
```

Comment:
Missed this one. The `||` should be an `&&`

Reply:
ah yes, thanks! will fix.